ModularTegustation/lc13_effects.dm

```diff
@@ -20,3 +20,15 @@
 	pixel_x = base_pixel_x
 	pixel_y = base_pixel_y
 	animate(src, alpha = 0, time = 13)
+
+/obj/effect/temp_visual/workcomplete  // Work complete effect
+	name = "work complete"
+	duration = 15
+	icon = 'icons/effects/160x96.dmi'
+	icon_state = "normal"
+	layer = ABOVE_ALL_MOB_LAYER
+
+/obj/effect/temp_visual/workcomplete/Initialize(mapload, set_dir)
+	. = ..()
+	animate(src, alpha = 0, time = 15)
```

Comment:
I think it would look better if it started at ~125 alpha and "swayed" from +20 to -20 for its duration before fading out.

Reply:
At such a small change the effect is not very noticeable, but I did take aspects of your suggestion into consideration.